kitsune/messages/jinja2/messages/outbox.html

```diff
@@ -14,81 +14,60 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
                     <a rel="nofollow" href="{{ profile_url(user) }}">
                       {%- if name -%}
                         {{ name }}
                       {%- else -%}
                         {{ user.profile.display_name }}
                       {%- endif -%}
-                    {%- if message.recipients_count > 1 -%}, ...{% endif %}
                     </a>
-                  {% else %}
+                    {%- if message.recipients_count > 1 -%}, ...{% endif %}
+                {% else %}
                    {{ _('None') }}
-                  {%- endfor %}
-                  </span>
-                  {% if in_staff_group(request.user) %}
-                    <span class="to group">
-                        {%- for group in message.to_groups[:1] -%}
-                          {%- for profile in group.profile.all() -%}
-                            <a href="{{ url('groups.profile', profile.slug) }}">{{ group }}</a>
-                          {%- endfor -%}
-                          {% if message.to_groups_count > 1 %}, ...{% endif %}
-                        {% else %}
-                          {{ _('None') }}
-                        {% endfor %}
-                    </span>
-                  {% endif %}
-                <span class="message-view">
-                  <a class="read message text-body-sm" href="{{ url('messages.read_outbox', message.id) }}">
-                    {{ message.content_parsed|striptags|truncate(length=160) }}
-                  </a>
-                </span>
-                <span class="delete-button">
-                 <a class="delete" href="{{ url('messages.delete_outbox', message.id) }}" title="{{ _('Delete message') }}">&#x2716;</a>
-                </span>
+                {%- endfor %}
                 </div>
-              </li>
+                {% if in_staff_group(request.user) %}
+                    <div class="email-cell to-groups">
+                    {%- for group in message.to_groups[:1] -%}
+                        {%- for profile in group.profile.all() -%}
+                        {% set group_slug = profile.slug %}
+                        <a href="{{ url('groups.profile', group_slug) }}">{{ group }}</a>
+                        {%- endfor -%}
+                        {% if message.to_groups_count>1 %}, ...{% endif %}
+                    {% else %}
+                        {{ _('None') }}
+                    {% endfor %}
+                    </div>
+                {% endif %}
+                <div class="email-cell excerpt">
+                    <a class="read message text-body-sm" href="{{ url('messages.read_outbox', message.id) }}">
+                        {{ message.content_parsed|striptags|truncate(length=160) }}
+                    </a>
+                </div>
+                <div class="email-cell delete">
+                  <a class="delete" href="{{ url('messages.delete', message.id) }}" title="{{ _('Delete message') }}">&#x2716;</a>
```

Comment:
This should be `'message.delete_outbox'`.

Reply:
`messages.delete_outbox` - Fixed!